1.1.0 changes

cs-abhishek-shukla opened the pull request:
- Removed all other content except readme and release notes

cs-rohit-khune commented on README.md:
```diff
@@ -19,10 +19,8 @@ The guide creates a task list and tracks to streamline multiple tasks such as:
 - Enabling purging of audit and playbook logs
-- Configuring enrichment
-- Configuring mitigation playbooks
+- Setup indicator extraction and exclusion
```
FortiSOAR Compatibility should be 7.6.1 and later

cs-rohit-khune commented on README.md:
```diff
@@ -19,10 +19,8 @@ The guide creates a task list and tracks to streamline multiple tasks such as:
-
-| [Installation](./docs/setup.md#installation) | [Configuration](./docs/setup.md#configuration) | [Usage](./docs/usage.md) |
-|----------------------------------------------|------------------------------------------------|--------------------------|
+Head over to the SOAR Framework solution pack documentation to track setup tasks using the **Setup Guide**!
```
we should give the link for SFSP documentation

cs-abhishek-shukla committed:
1.1.0 changes
- Removed all other content except readme and release notes
- Feedback comments addressed

cs-rohit-khune reviewed (approved):
LGTM
